public/app/features/dashboard-scene/settings/AnnotationsEditView.tsx

```diff
@@ -36,72 +45,210 @@ export class AnnotationsEditView extends SceneObjectBase<AnnotationsEditViewStat
     return getDashboardSceneFor(this);
   }
 
-  private get _dataLayers(): SceneDataLayerProvider[] {
-    return sceneGraph.getDataLayers(this._dashboard);
-  }
-
-  public getSceneDataLayers(): SceneDataLayers | undefined {
+  public getSceneDataLayers(): SceneDataLayers {
```

Comment:
Can we move this util to `dashboardSceneGraph.getDataLayers`? again - to handle the potential structural changes in the future.